scripts/migrations/migrate_2.2.1.js

```diff
@@ -14,10 +13,14 @@ const config = {
     "SellerHandlerFacet",
     "DisputeResolverHandlerFacet",
     "OrchestrationHandlerFacet1",
+    "ProtocolInitializationHandlerFacet",
   ],
   remove: [],
   skipSelectors: {},
-  facetsToInit: {},
+  facetsToInit: {
```

Comment:
There are two things:
1. first we have "initialized interfaces", which are stored in `ps.initializedInterfaces` and are used by the `onlyUninitialized` modifier. This value can currently be set to true only by calling `initialize`, that's why I added them here. If I didn't, well, I don't think anything bad could happen. It's more of a consistency thing. This value is a bit peculiar since it can only be turned to true and not back to false (currently). I'm happy to discuss if we even need this initialization pattern or if should we simply deprecate it. Since initialize should never be registered as a diamond method, I don't see a security risk there if we omit `onlyUninitialized` from facets in the upcoming releases.
2. Then we also have "supported Interfaces" which are stored in `ds.supportedInterfaces` and are used when someone queries `erc165.supportsInterface`. The upgrade script already passes a list of `interfaceIds` to add and remove to `ProtocolInitializationHandlerFacet` when the diamond cut is made. So no need to do any changes to those scripts. But yeah, it might be worth adding that to generic upgrade tests, since I assume most upgrades will change at least some interfaces.

Reply:
IMO modifiers should be used for security reasons and if this modifier is not prevent nothing bad from happen I think we should remove

Reply:
added an issue just to not ended up loosing this discussion as I'm going to merge this PR